lib/render.js

```diff
@@ -340,11 +342,11 @@ function renderExpressRoute(req, res, next) {
       const diff = process.hrtime(hrStart),
         ms = Math.floor((diff[0] * 1e9 + diff[1]) / 1000000);
 
-      log('info', 'rendered express route:' + (req.hostname + req.baseUrl + req.path) + ' ' + ms + 'ms');
+      log('info', `rendered express route: ${req.hostname + req.baseUrl + req.path} ${ms}ms`);
     }
   }).catch((error) => {
     if (error.name === 'NotFoundError') {
-      log('verbose', 'in', req.uri, error.message);
+      log('error', `in ${req.uri}: ${error.message}`);
```

Comment:
does error.message tell me it's not found? or is that only in error.name?

Reply:
It will tell you it's not found